benchmarks/benchmarks/bench_core.py

```diff
@@ -99,6 +101,12 @@ def time_triu_indices_500(self):
     def time_tril_indices_500(self):
         np.tril_indices(500)
 
+    def time_mean_small_array(self):
+        np.mean(self.array_5)
+
+    def time_mean_axis(self):
+        np.mean(self.array_4_5, axis=1)
+
 
```

Comment:
It turns out we do not bencmark `np.mean`. I think this belongs in `bench_functions_base.py`, since that is where `median`, `percentile` and other numeric functions are checked.